### PR TITLE
ziggurat: linking state merge

### PR DESCRIPTION
**Pull request by hosted-fornet**
**Problem**:

1. State is not properly propagated for linked projects.
2. Machine-generated test-steps are bad to show on FE.

**Solution**:

1. Properly propagate.
2. Delete test-steps after setup.

**Notes**:

FYI @tadad

### CODE REVIEW

**Comment by hosted-fornet** [comment]
Whoops, this is not quite working right.

**Comment by hosted-fornet** [comment]
Okay fixed :+1:

File: lib/zig/ziggurat.hoon [hunk 1]
```diff
@@ -1181,11 +1174,14 @@
...
-  ^-  inflated-state-0:zig
+  ^-  _state
+  ::  linked-projects is, at a minimum, project-name
   =/  linked-projects=(list @t)
-    ~(tap in (~(get ju linked-projects.state) project-name))
+    %~  tap  in
+    %.  project-name
+    ~(put in (~(get ju linked-projects.state) project-name))
   |-
-  ?~  linked-projects  ~&(%z^%cslp^%final^(~(run by projects.state) |=(p=project:zig pyro-ships.p)) state)
```
Comment (0x70b1a5) [inline lib/zig/ziggurat.hoon:1188]: I'm glad this is gone, whatever it is, lol

**Review by 0x70b1a5 — APPROVED** [review]
LGTM 🦧

File: lib/zig/ziggurat.hoon (hunk 1 above)
Comment (hosted-fornet) [reply lib/zig/ziggurat.hoon:1188]: Just a leftover debugging ~&. It is a bit spicy, though :hot_pepper:

**Pull request merged**